Getting-Started-with-Akka-http-signature.md

```diff
@@ -0,0 +1,250 @@
++Hz2EGrkpkM0Zr6yHfEmW/H8mrnMgNev+MxND2euKGR+QEjajfdTSTuN5qjglhQWtLN/cBKQfS51
+YDS4qExuSVyPHYH2mEOuUTeVcdg7OOHAigjDdI73XtfssBbQyEJrMMjFBgoI+H9nZLDsFGZ81vDa
+oSRBVwIDAQAB"""
+```
+
+All keys can be converted from a PEM formatted String back to decimal format. The readPublicKeyFrom() function takes a String in Base64 notation and returns the original value of the key. This means that a public key, saved to a file in PEM format can always be converted back to it's original format by the user at a later point in time.
+
+```scala
+@ val publ = RSAKeys.readPublicKeyFrom(basePublic)
+
+publ: scala.util.Try[java.security.PublicKey] = Success(
+  Sun RSA public key, 2048 bits
+  modulus: 18617616591683306896114159596818930151392321021655743092554571476906272216221226120621265951005438129779697999635487320523329319543618461573656972895570742121788481588357965539288547158041270238222755734178260848243117538931191184092136701393910768801017892322044665470799614169981926296276068464740129396666480935344897491410772746293264767223181696074228079861695236831569819678997680346803798366156295040872828579274684209593397084757356046266872087618604860475724560124477904326733099073845912710038077198861072318221134249054814455997792289530305256172821638522445655947916737229103835394051258962680413560258903
+  public exponent: 65537
+)
+```
+The .readPublicKeyFrom() function returns a Try of Public/Private key depending on whether the String given can be parsed back to some valid key. If the operation is successful the value of the original key is assigned to the new value in BigInt format.
+
+The user can then save his keys on his local filesystem by using the following ammonite commands:
+
+```scala
+write(wd/"publicKey.pem", RSAKeys.save(pub))
+write(wd/"privateKey.pem", RSAKeys.save(priv))
+```
+This will save the contents of the key in String format within the file found by following the specified path in .pem files.
+
+### Saving a public Key to RDF
+
+A user can also transform his public keys to an RDF. In order to do this one must first import multiple files in order to resolve the required dependencies.
+
+The org.w3 declarations required are:
+
+```scala
+import org.w3.banana.binder
+import org.w3.banana.binder.RecordBinder
+implicit val binder = RecordBinder
+val cert = CertPrefix[Rdf]
+```
+Several java imports from java's security and math library are also required for this process:
+
+```scala
+import java.math.BigInteger
+import java.security.KeyFactory
+import java.security.interfaces.RSAPublicKey
+import java.security.spec.RSAPublicKeySpec
+```
+
+After that the binder and all it's functionalities can be imported:
+
+```scala
+import org.w3.banana.binder._
+import recordBinder._
+```
+
+Finally, after that, the user can create the Cert object that contains the required dependencies to turn a Key into a Pointed Graph
+
+```scala
+object Cert {
+
+          implicit val rsaClassUri = classUrisFor[RSAPublicKey](cert.RSAPublicKey)
```

Comment:
way too much white space in this whole code snippet. Please put 3 white spaces for indentation.